src/components/NewAppModal.js

```diff
@@ -45,7 +45,7 @@ export const NewAppModal = _.flow(
     Utils.withBusyState(setLoading),
     withErrorReporting('Error deleting app')
   )(async () => {
-    await Ajax().Apps.app(app.googleProject, app.appName).delete()
+    await Ajax().Apps.app(app.googleProject, app.appName).delete(true)
```

Comment:
Since we always intend to send `true` for now, perhaps we should hard code that in the ajax call.

Reply:
Sure. I can do that.